### PR TITLE
[KOGITO-152] - Upgrading images to 0.3 and support for custom image input on CRD

### PR DESCRIPTION
**Pull request by ricardozanini**
See:
https://issues.jboss.org/browse/KOGITO-152

Besides updating the images to 0.3, we also added support for users to inform a custom image tag, this way will be possible for users to use old or new images with the Operator.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: deploy/catalog_resources/courier/kiecloud-operatorsource.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorSource
 metadata:
-  name: kiecloud-operators
+  name: kogitocloud-operator
```
Comment (xiezhang7) [inline deploy/catalog_resources/courier/kiecloud-operatorsource.yaml]: The file should be renamed, too.

Comment (ricardozanini) [reply deploy/catalog_resources/courier/kiecloud-operatorsource.yaml]: Yeah... I was thinking about it... I don't know why I didn't rename it :-1:

File: README.md [hunk 2]
```diff
@@ -85,60 +85,44 @@ This function will create a new `RoleBinding` that depends on the `ServiceAccoun
...
 
 ```bash
-make
+$ make
 ```
 
 ## Installation
 
-Kogito Operator is not available in the OperatorHub [yet](https://issues.jboss.org/browse/KOGITO-67), hence have to be installed manually on [OpenShift 4.x](#deploy-to-openshift-4.x-using-olm) or [OpenShift 3.11](#deploy-to-openshift-311-manually).
+Kogito Operator is not available in the OperatorHub [yet](https://issues.jboss.org/browse/KOGITO-67), hence have to be installed manually on [OpenShift 4.x](#deploy-to-openshift-4.x) or [OpenShift 3.11](#deploy-to-openshift-311-manually).
```
Comment (mramendi) [inline README.md]: "has to be"

**Comment by ricardozanini** [comment]
@mramendi we're going to merge this to run the final tests for tomorrow release. Could you kindly review (if you have the time) the README.md file and open a PR for your suggestions, please? Many thanks!

File: README.md [hunk 3]
```diff
@@ -85,60 +85,44 @@ This function will create a new `RoleBinding` that depends on the `ServiceAccoun
...
 ```
 
-If pushing to another quay repository, replace _kiegroup_ with your username or other namespace. Notice that the push command does not overwrite an existing repository, and the bundle needs to be deleted before a new version can be built and uploaded. Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster) to load your operator bundle in OpenShift.
+It will take a few minutes for the operator to become visible under the _OperatorHub_ section of the OpenShift console _Catalog_. The Operator can be easily found by filtering by _Kogito_ name.
 
-Note that the OpenShift cluster needs access to the created application. Make sure that the application is **public** or you have configured the private repository credentials in the cluster. To make the application public, go to your quay.io account, and in the _Applications_ tab look for the `kogitocloud-operator` application. Under the settings section click on _make public_ button.
+Verify if the Operator status by running:
```
Comment (mramendi) [inline README.md]: "Verity if the operator status" does not make sense, please work out the actual requirement. Verify operator status? Verify availability of the operator>

File: README.md [hunk 4]
```diff
@@ -150,28 +134,27 @@ $ oc create -f deploy/crs/app_v1alpha1_kogitoapp_cr.yaml
 kogitoapp.app.kiegroup.org/example-quarkus created
 ```
 
-### Clean up a KogitoApp deployment
+Or you can always use the [CLI](#cli) to deploy your services.
+
+To look at the Operator logs, first identify the pod where it's deployed:
```
Comment (mramendi) [inline README.md]: "Where the Operator is deployed" probably?

File: README.md (hunk 3 above)
Comment (ricardozanini) [reply README.md]: I guess that "Verify operator status" sounds better, plus makes more sense for this context.

**Pull request merged**